tests/nightly/test_np_large_array.py

```diff
@@ -696,16 +696,16 @@ def test_one_hot():
     assert A.grad[0] == 0
 
 @use_np
-@pytest.mark.skip(reason='backward value broken on large tensor')
 def test_pick():
     A = np.zeros((INT_OVERFLOW, 2))
     B = np.zeros((INT_OVERFLOW))
+    A[0, 0] = 3
```

Comment:
Can you pick a very large index instead of 0 ?

Reply:
sure I can also check the value of a large index.
However because https://github.com/apache/incubator-mxnet/issues/19014 this issue (assigment operator bug) we cannot modify the value of a large index

Reply:
Since the operator is `pick` and it returns values of indices provided in second argument. I wouldn't say its fixed then because you are unable to access large indices by passing them as argument. You should perhaps look into `operator=` implementation of this. Let me know if you think thats not that case or if i could explain it a bit more clearly

Reply:
Accessing the element actually works fine. But I totally agree with you in that to be 100% confident, I can fix assignment operator first and then try this test again by setting a value of large index to something and then picking it

Reply:
In that case I can prioritize assignment operator next week and come back to this pr in a few days

Reply:
@Zha0q1 #19014 has been merged now